utils/display.go

```diff
@@ -9,23 +9,41 @@ under the terms of the MIT License; see LICENSE file for more details.
 package utils
 
 import (
+	"encoding/json"
+	"fmt"
 	"os"
 
 	"github.com/jedib0t/go-pretty/v6/table"
 )
 
-func DisplayTable(header []interface{}, rows [][]interface{}) {
+func DisplayTable(header []string, rows [][]any) {
+	// Convert to table.Row type
 	rowList := make([]table.Row, len(rows))
 	for i, r := range rows {
 		rowList[i] = r
 	}
 
+	headerRow := make(table.Row, len(header))
+	for i, h := range header {
+		headerRow[i] = h
+	}
+
 	t := table.NewWriter()
 	t.SetOutputMirror(os.Stdout)
-	t.AppendHeader(header)
+	t.AppendHeader(headerRow)
 	t.AppendRows(rowList)
 	t.Style().Options.DrawBorder = false
 	t.Style().Options.SeparateColumns = false
 	t.Style().Options.SeparateHeader = false
 	t.Render()
 }
+
+func DisplayJsonOutput(output any) {
+	byteArray, err := json.MarshalIndent(output, "", "  ")
+
+	if err != nil {
+		fmt.Println("Error: ", err)
+	}
```

Comment:
We should either call `os.Exit(1)` or return the error and handle it in `displayListPayload`. What do yoy think?

Reply:
I think we should call `os.Exit(1)` since `displayListPayload` (and the other functions) should always call this with the right arguments. It's also what's being done in other places, #16 is meant to improve this behaviour everywhere